src/components/AnacondaWizard.jsx

```diff
@@ -172,90 +178,97 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, runtim
         },
     ];
 
+    const componentProps = {
+        isFormDisabled,
+        onCritFail,
+        setIsFormDisabled,
+        setIsFormValid,
+    };
+
     const getFlattenedStepsIds = (steps) => {
         const stepIds = [];
         for (const step of steps) {
+            stepIds.push(step.id);
             if (step.steps) {
                 for (const childStep of step.steps) {
                     if (childStep?.isHidden !== true) {
                         stepIds.push(childStep.id);
                     }
                 }
-            } else {
-                stepIds.push(step.id);
             }
         }
         return stepIds;
     };
     const flattenedStepsIds = getFlattenedStepsIds(stepsOrder);
-
-    const { path } = usePageLocation();
     const firstStepId = stepsOrder.filter(step => !step.isHidden)[0].id;
-    const currentStepId = path[0] || firstStepId;
 
     const isStepFollowedBy = (earlierStepId, laterStepId) => {
         const earlierStepIdx = flattenedStepsIds.findIndex(s => s === earlierStepId);
         const laterStepIdx = flattenedStepsIds.findIndex(s => s === laterStepId);
-        return earlierStepIdx < laterStepIdx;
-    };
-
-    const canJumpToStep = (stepId, currentStepId) => {
-        return stepId === currentStepId || isStepFollowedBy(stepId, currentStepId);
+        if (laterStepIdx === -1) {
```

Comment:
If I understand it right: "any step is followed by a step which is not found" it seems a bit counterintuitive to me for the function name (and original purpose).

Reply:
I found a solution where we don't have to change the isStepFollowedBy function.